dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -179,7 +179,19 @@ public void IsCloseableTest()
                 Wait.ForIdle();
 
                 ElementCache.Refresh();
+
+                Log.Comment("First close button should be visible because IsCloseable was set to true");
                 closeButton = FindCloseButton(firstTab);
+                Verify.IsNotNull(closeButton);
+
+                UIObject tab = FindElement.ByName("SecondTab");
+                Log.Comment("Second close button should be visible because IsCloseable was set to true in xaml");
+                closeButton = FindCloseButton(tab);
+                Verify.IsNotNull(closeButton);
+
+                tab = FindElement.ByName("LongHeaderTab");
+                Log.Comment("Third close button should be visible because IsCloseable is still unset");
```

Comment:
Typo, I believe: "should NOT be visible"

Reply:
At least that's what the `Verify` check seems to imply.

Reply:
oops -- yes.